test/Sentry.Maui.Device.TestApp/Startup.cs

```diff
@@ -30,6 +32,19 @@ public static MauiApp CreateMauiApp()
                 conf.AddXunit();
             });
 
+        //             .UseVisualTestRunner(conf =>
```

Comment:
Should this be commented out?

Reply:
Yes - It caused issues for CI.  We definitely need it when debugging unit tests in MAUI though.  This was a god send today

Reply:
When will/would we uncomment it? When running locally? If so, we need to work out how to install the necessary dependencies to get this working on CI. 

Is there any way we can do that via directives instead?

Reply:
Updated from gestures branch that has all of the flags we need to make this proper/better